bosh/releases/pre_render_scripts/diego-api/cfdot/jobs/patch_pre-start.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+##
+# ATTENTION: This is part of a set of six interconnected patches.
+#            Two files spread over three instance groups.
```

Comment:
I wish these patches were somehow implemented with a template helper function, so we don't have almost identical code in 3 different places. But this is also true for other patches we have, e.g. the buildpacks, that move files to `/var/vcap/data/shared-packages`. So maybe we can consolidate all this later (hah) in a future cleanup PR.

Reply:
Leaving it to later PR.